src/components/map.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+import React from 'react';
+import ReactMapboxGl, { Layer, Feature } from 'react-mapbox-gl';
+import 'mapbox-gl/dist/mapbox-gl.css';
+
+const Map = ReactMapboxGl({
+  accessToken: 'pk.eyJ1IjoidGhlZ2l2aW5naGVhcnQiLCJhIjoiY2xwNWl3bm50MWVudzJqbzRzMHQ4cHRpdyJ9.hd8bhxh4uUXhe1xA1X9tCA'
```

Comment:
should this be public or put into the environment variables?

Reply:
It was in .env.local initially but I thought it was the generic public one so it was fine. Now realizing that's prob not the case, I've put it back

Reply:
I've added the env variables to the resources section of Notion